src/v/serde/parquet/column_writer.cc

```diff
@@ -249,7 +237,8 @@ class buffered_column_writer final : public column_writer::impl {
 private:
     column_stats_collector<value_type, comparator> _current_page_stats;
     column_stats_collector<value_type, comparator> _flushed_stats;
-    size_t _current_buffer_size = 0;
+    int64_t _page_memory_usage = 0;
```

Comment:
nit on naming:  consider _page_memory_usage -> _current_page_memory_usage. 

_page_memory_usage givens an impression that it accounts for all pages 

.. or may be just a comment is ok too.

Reply:
There is going to be a follow here pending some perf investigation/followups. Will update this there.